commons-geometry-examples/examples-io/src/main/java/org/apache/commons/geometry/examples/io/threed/AbstractModelIOHandler.java

```diff
@@ -35,10 +35,10 @@
     public BoundarySource3D read(final String type, final File in, final DoublePrecisionContext precision) {
         ensureTypeSupported(type);
         try {
-            try (InputStream is = Files.newInputStream(in.toPath())) {
+            try (final InputStream is = Files.newInputStream(in.toPath())) {
```

Comment:
These `final` modifiers on try-with-resources statements are redundant and cause checkstyle failures. You can see these if you run maven with the `mvn clean install -Pcommons-geometry-examples` so that the example modules are included in the build.